### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by VixieTSQ** [swcolegrove/css-override-web-extension#3]
This project is **not FLOSS.** It is only source available. Please add a license. My personal recommendation is AGPL as it classifies use over a network as distribution.

**Comment by 12joan** [comment]
I agree that a license should be added to this GitHub Repo. (My recommendation is Unlicense or MIT, since they make other developers' lives easier without restricting what those developers are allowed to do with their own code, as GPL-family licenses do.)

For anyone wanting to fork this extension, note that most of the source files (all of the important ones) are already [released on Mozilla](https://addons.mozilla.org/en-US/firefox/addon/css-override/) under [MPL-2.0](https://www.mozilla.org/en-US/MPL/2.0/). To see which files are covered under this license, download the XPI file for the extension and unzip it as if it were a ZIP file.

https://addons.mozilla.org/firefox/downloads/file/3419738/css_override-1.0.xpi

Note also that the license field in package.json is listed as ISC. While this license field isn't [usually considered](https://opensource.stackexchange.com/a/7244) enough to grant a license by itself, it gives us a hint as to what rights @swcolegrove intended to grant when publishing the source code.

**Comment by djragsdale** [comment]
As ISC is the npm default, I would hardly count it as a hint to an intention. That said, fork it if you want. @swcolegrove isn't going to sue you. He just made this extension so he could fix bad Jira styles.
